### PR TITLE
Rdm 5326

### PR DESCRIPTION
**Pull request by akdasari**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # https://help.github.com/en/articles/about-code-owners
 # Documentation: https://github.com/hmcts/cnp-flux-config/blob/master/CODEOWNERS.md
 
-CODEOWNERS @hmcts/platform-engineering @hmcts/devops
+CODEOWNERS @hmcts/platform-engineering @hmcts/devops @hmcts/ccd-demo
```
Comment (timja) [inline CODEOWNERS]: don't do this, just add one for ccd-demo

File: CODEOWNERS [hunk 2]
```diff
@@ -35,3 +35,6 @@ k8s/**/cluster-*/static-overlay/static/kustomization.yaml @hmcts/platform-engine
 k8s/**/cluster-*/static-overlay/static/.flux.yaml @hmcts/platform-engineering
 
 ### Next team
+
+##ccd-demo
+k8s/**/common/ccd/ccd-demo.yaml @hmcts/ccd-demo
```
Comment (timja) [inline CODEOWNERS]: don't wildcard the beginning, provide the full path,
```suggestion
k8s/demo/common/ccd/*.yaml @hmcts/ccd-demo
```

File: CODEOWNERS (hunk 1 above)
Comment (timja) [reply CODEOWNERS]:
```suggestion
CODEOWNERS @hmcts/platform-engineering @hmcts/devops
```

Comment (akdasari) [reply CODEOWNERS]: deleted

File: CODEOWNERS (hunk 2 above)
Comment (akdasari) [reply CODEOWNERS]: modified as suggested

**Pull request merged**